rpa-builder/6.0/modules/ROOT/pages/using-workflow-debugger.adoc

```diff
@@ -0,0 +1,57 @@
+= Using Workflow Debugger
+
+RPA Builder provides an integrated debugger for Workflows. To use it, click the *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) or press _F8_. This starts a test run and opens Workflow Debugger instead of stopping the execution of the Workflow if an error occurs.
+
+When you execute a *Debug Workflow* run, you can call Workflow Debugger by pressing _Pause/Break_ on your keyboard. You can change the configuration of Workflow objects within the Workflow Debugger and then re-run them or continue from the current position. When the Workflow enters an error state, Workflow Debugger shows.
+
+image:workflow-debugger-window.png[The Workflow Debugger window]
+
+<1> The first pane in Workflow Debugger shows a condensed view of your Workflow and an arrow indicating which step was the last to execute.
+<2> On the second pane is the debug information for the selected Action Step. The Debug Information panel contains a combination of configuration settings and runtime information relating to Action Steps already executed.
+
+In this example, the Workflow entered the error state at the Window Wait Action Step. In the Debug Information panel, the *Result* section confirms that the window could not be found. Also, in the *Configuration* section, the *WindowTitle* value is set to `testname`, which causes the error because no window with that title exists during the execution.
```

Comment:
```suggestion
In this example, the Workflow entered the error state at the Window Wait Action Step. In the Debug Information panel, the *Result* section confirms that the window could not be found. Also, in the *Configuration* section, the *WindowTitle* value is set to `testname`, which caused the error because no window with that title existed during the execution.
```

Reply:
I got a little hung up on the verb tenses here.